test/e2e/specs/blocks/shared/block-smoke-testing.ts

```diff
@@ -53,7 +53,8 @@ export function createBlockTests( specName: string, blockFlows: BlockFlow[] ): v
 				it( `${ blockFlow.blockSidebarName }: Add the block from the sidebar`, async function () {
 					await editorPage.addBlockFromSidebar(
 						blockFlow.blockSidebarName,
-						blockFlow.blockEditorSelector
+						blockFlow.blockEditorSelector,
+						{ noSearch: true }
```

Comment:
The sidebar block inserter is very slow, especially when searching.

As there exists other specs covering the search and add portion, I've decided that speeding up this spec was preferable.